neurom/view/tests/test_view.py

```diff
@@ -46,12 +47,19 @@
 
 
 def test_tree():
-    with get_fig_2d() as (fig, ax):
-        tree = fst_neuron.neurites[0]
-        view.plot_tree(ax, tree, color='black', diameter_scale=None, alpha=1., linewidth=1.2)
-        collection = ax.collections[0]
-        nt.eq_(collection.get_linewidth()[0], 1.2)
-        np.testing.assert_allclose(collection.get_color(), np.array([[0., 0., 0., 1.]]))
+    neuron = load_neuron(os.path.join(SWC_PATH, 'simple-different-section-types.swc'))
+    expected_colors = {'black': np.array([[0., 0., 0., 1.] for _ in range(3)]),
```

Comment:
What about `np.repeat([[0,0,0,1]], 3, axis=0)`?

Reply:
I find this more easily understandable than having to reason about numpy axes.